go/viperutil/funcs/decode.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package funcs
+
+// TODO: this creates an import cycle ... IMO tabletenv should not define Seconds,
+// it should be in something more akin to flagutil with the other values like
+// TabletTypeFlag and friends.
```

Comment:
Agreed

Reply:
Going to file an issue to follow up on this one, as it'll involve some more moving stuff around

Reply:
https://github.com/vitessio/vitess/issues/11891